config.go

```diff
@@ -10,110 +10,151 @@ const (
+	RemoteServerIPConfigKey             = "remote_server_ip" // TODO: remove
+	UserAgentConfigKey                  = "user_agent"
 
-	// todo should use this config value to implement cross-platform currency conversion feature
-	CurrencyConversionConfigKey = "currency_conversion_option"
+	LastTxHashConfigKey = "last_tx_hash"
 
 	VSPHostConfigKey = "vsp_host"
 )
 
-func (lw *LibWallet) SaveUserConfigValue(key string, value interface{}) error {
-	return lw.configDB.Set(userConfigBucketName, key, value)
+func (mw *MultiWallet) SaveUserConfigValue(key string, value interface{}) error {
+	return mw.configDB.Set(userConfigBucketName, key, value)
 }
 
-func (lw *LibWallet) ReadUserConfigValue(key string, valueOut interface{}) error {
-	err := lw.configDB.Get(userConfigBucketName, key, valueOut)
-	if err != nil && err != storm.ErrNotFound {
+func (mw *MultiWallet) ReadUserConfigValue(key string, valueOut interface{}) error {
+	err := mw.configDB.Get(userConfigBucketName, key, valueOut)
+	if err != nil {
 		return err
 	}
 	return nil
 }
 
-func (lw *LibWallet) SetBoolConfigValueForKey(key string, value bool) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetBoolConfigValueForKey(key string, value bool) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) SetDoubleConfigValueForKey(key string, value float64) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetDoubleConfigValueForKey(key string, value float64) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) SetIntConfigValueForKey(key string, value int) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetIntConfigValueForKey(key string, value int) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) SetLongConfigValueForKey(key string, value int64) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetInt32ConfigValueForKey(key string, value int32) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) SetStringConfigValueForKey(key, value string) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetLongConfigValueForKey(key string, value int64) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) ReadBoolConfigValueForKey(key string) (valueOut bool) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
+func (mw *MultiWallet) SetStringConfigValueForKey(key, value string) {
+	err := mw.SaveUserConfigValue(key, value)
+	if err != nil {
+		log.Errorf("error setting config value: %v", err)
```

Comment:
Modify `mw.SaveUserConfigValue` to log errors. That way, you won't need to perform this logging in each of the helper functions. Also, if possible, you could modify the `mw.SaveUserConfigValue` method to not return anything.